Week3/homework/7-step3.js

```diff
@@ -22,4 +22,15 @@ f2(y);
 console.log(y);
 
 // Add your explanation as a comment here
-
+/*
+When passing in a primitive type variable like a string or a number,
+the value is passed in by value.This means that any changes to that variable
+while in the function are completely separate from anything that happens outside the function.
+*/
+
+/*
+Passing in an object, however,
+passes it in by reference. In this case, any property of that object is accessible within the function
+*/
+
+//via https://snook.ca/archives/javascript/javascript_pass
```

Comment:
I applaud that you included a link to the source where found this information. I assume you also understand what it stated here.